### PR TITLE
web app sample

### PR DESCRIPTION
**Pull request by SomkaPe**

### CODE REVIEW

File: src/samples/msal-web-sample/src/main/java/com/microsoft/azure/msalwebsample/User.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    protected String userPrincipalName;
+
+    @Override
+    public String toString() {
+        return new JSONObject(this).toString();
+    }
+}
+
+
+
```
Comment (sangonzal) [inline src/samples/msal-web-sample/src/main/java/com/microsoft/azure/msalwebsample/User.java]: nit: extra lines

File: src/samples/msal-web-sample/src/main/resources/application.properties [hunk 2]
```diff
@@ -0,0 +1,4 @@
+aad.clientId=
+aad.authority=
+aad.secretKey=
+aad.redirectUri=
```
Comment (sangonzal) [inline src/samples/msal-web-sample/src/main/resources/application.properties]: As discussed, might make sense to include default redirectUri

File: src/samples/msal-web-sample/src/main/java/com/microsoft/azure/msalwebsample/BasicConfiguration.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+@Setter
+@Component
+@ConfigurationProperties("aad")
+class BasicConfiguration {
+    String clientId;
+    @Getter(AccessLevel.NONE) String authority;
+    String redirectUri;
+    String secretKey;
+
+    String oboClientId;
```
[the remark this answers is not in the document]
Comment (SomkaPe) [reply src/samples/msal-web-sample/src/main/java/com/microsoft/azure/msalwebsample/BasicConfiguration.java]: removed

File: src/samples/msal-web-sample/src/main/java/com/microsoft/azure/msalwebsample/HttpClientHelper.java [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+
```
[the remark this answers is not in the document]
Comment (SomkaPe) [reply src/samples/msal-web-sample/src/main/java/com/microsoft/azure/msalwebsample/HttpClientHelper.java]: removed

File: src/samples/msal-web-sample/src/main/java/com/microsoft/azure/msalwebsample/User.java (hunk 1 above)
Comment (SomkaPe) [reply src/samples/msal-web-sample/src/main/java/com/microsoft/azure/msalwebsample/User.java]: removed

File: src/samples/msal-web-sample/src/main/resources/application.properties (hunk 2 above)
Comment (SomkaPe) [reply src/samples/msal-web-sample/src/main/resources/application.properties]: added

**Pull request merged**